package.json

```diff
@@ -13,6 +13,7 @@
         "cypress-react-selector": "^2.3.17",
         "cypress-xpath": "^1.6.2",
         "faker": "^5.5.3",
+        "js-yaml": "^4.1.0",
```

Comment:
I think the CI is failing because the package-lock.json wasn't updated to reflect this change